### PR TITLE
Explore nearby

### PR DESCRIPTION
**Pull request by akami**
this will be a very beautiful merge conflict ;)

### CODE REVIEW

File: app/constants/Styles.tsx [hunk 1]
```diff
@@ -1,13 +1,49 @@
...
+        alignItems: 'center',
+
+        width: 350,
+        padding: 8,
+
+        marginVertical: 16,
+        marginHorizontal: 16,
+
+        borderWidth: 1,
+        borderColor: 'gray',
```
Comment (samox73) [inline app/constants/Styles.tsx:32]: I'd say that we should not introduce color specifications in Stylex.tsx directly. To be more flexible, especially for theming support, defining the needed colors in Colors.tsx should both provide a better overview and be easier to use.

File: app/constants/Styles.tsx [hunk 2]
```diff
@@ -1,13 +1,49 @@
...
+        borderRadius: 15,
+
+        backgroundColor: 'white'
```
Comment (samox73) [inline app/constants/Styles.tsx:35]: Same as above.

**Review by samox73 — APPROVED** [review]
Hallo?

**Comment by akami** [comment]
> Hallo?

Hallo?

**Pull request merged**